tests/main/searching/task.yaml

```diff
@@ -1,8 +1,11 @@
 summary: Check snap search
 
 execute: |
-    echo "Try to list all snaps"
-    ( snap find 2>&1 || echo FAILED ) | grep -Pzq "(?ms)you need to specify a query.*FAILED"
+    echo "List all featured snaps"
+    expected="(?s)Name +Version +Developer +Notes +Summary *\n\
+    (.*?\n)?\
+    .*"
+    snap find | grep -Pzq "$expected"
```

Comment:
Can we test something we know will tend to remain featured here, so we have a hint that the feature in fact works?